### PR TITLE
Shipit integration

### PR DESCRIPTION
**Pull request by GoodForOneFare**
Allows publication via shipit.  Also adding misc configuration stuff for this standalone repo (let me know if you'd prefer that in a separate PR).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +23,10 @@
     }
   },
   "bin": "./bin/esify",
+  "devDependencies": {
+    "eslint": "3.10.x",
+    "eslint-plugin-shopify": "latest"
```
Comment (lemonmade) [inline package.json]: Can we pin down a version here? Safer that way.

File: package.json [hunk 2]
```diff
@@ -5,6 +5,10 @@
   "license": "MIT",
   "author": "Bouke van der Bijl <bouke@shopify.com>",
```
Comment (lemonmade) [inline package.json]: We should change this to be `Shopify <dev@shopify.com>` I think

File: package.json [hunk 3]
```diff
@@ -5,6 +5,10 @@
...
   "description": "A CLI for converting Shopify’s CoffeeScript to ESNext.",
+  "scripts": {
+    "deploy": "npm run lint && node ./script/deploy.js",
```
Comment (lemonmade) [inline package.json]: If there's no directory for scripts already, can we name it `scripts`? Plural makes more sense to my brain :D

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,5 @@
+node_modules
+reports
+coverage
```
Comment (lemonmade) [inline .gitignore]: `reports`/ `coverage` needed?

**Review by lemonmade — COMMENTED** [review]
A few minor comments, then looks good

File: .gitignore (hunk 4 above)
Comment (GoodForOneFare) [reply .gitignore]: Nope.  Removed from here and `.eslintignore`.

**Comment by GoodForOneFare** [comment]
Thanks!  Should be good to go, now.  Once I've merged this, I'll submit a PR to delete `Shopify/javascript/packages/esify`.

**Review by lemonmade — APPROVED** [review]

**Pull request merged**